packages/forklift-console-plugin/src/modules/Plans/views/details/components/PlanWarningCondition.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import Linkify from 'react-linkify';
+
+import { Alert, Text, TextContent, TextVariants } from '@patternfly/react-core';
+
+export const PlanWarningCondition: React.FC<{
```

Comment:
no need for React.FC , u can directly import FC

Reply:
More than half of our current code declare components like above, so if we want to use one format for all, I'll suggest to fix that in a follow up PR.